lambeq/__init__.py

```diff
@@ -113,7 +115,8 @@
 from lambeq.pregroups import (create_pregroup_diagram, diagram2str,
                               is_pregroup_diagram, remove_cups, remove_swaps)
 from lambeq.rewrite import (RewriteRule, CoordinationRewriteRule,
-                            CurryRewriteRule, SimpleRewriteRule, Rewriter)
+                            CurryRewriteRule, SimpleRewriteRule, Rewriter,
```

Comment:
Let's sort the imports alphabetically.

Reply:
So you mean like : 
CoordinationRewriteRule, CurryRewriteRule, DiagramRewriter, MergeWiresRewriter, Rewriter, RewriteRule, SimpleRewriteRule ?

Reply:
Imports are typically grouped together if they're subclasses of the same base class (multiple `RewriteRule`s for eg.), and then ordered alphabetically.

I'd recommend taking a look at some of the init files in lambeq, for eg. https://github.com/CQCL/lambeq/blob/main/lambeq/__init__.py